scripts/unassign-inactive-issues.js

```diff
@@ -0,0 +1,296 @@
+const core = require('@actions/core');
+const fetch = require('node-fetch-native');
+
+async function run() {
+  try {
+    const { Octokit } = await import('@octokit/rest');
+
+    const token = process.env.WEB_Token;
+    const slackWebhookUrl = process.env.SLACK_WEBHOOK_URL;
+    
+    if (!token) {
+      throw new Error('No authentication token provided. Please ensure WEB_Token is set in the workflow.');
+    }
+
+    if (!slackWebhookUrl) {
+      throw new Error('Slack webhook URL missing. Please ensure SLACK_WEBHOOK_URL is set in the workflow.');
+    }
+
+    console.log('Token exists:', !!token);
+    console.log('Slack webhook URL exists:', !!slackWebhookUrl);
+
+    const unassignments = [];
+
+    const inactivityPeriodInMinutes = 1;
+
+    const repository = process.env.GITHUB_REPOSITORY;
+    if (!repository) {
+      throw new Error('GITHUB_REPOSITORY environment variable is not set');
+    }
+
+    const [owner, repo] = repository.split('/');
+    console.log(`Processing repository: ${owner}/${repo}`);
+
+    // Function to send Slack notification via webhook
+
+    async function sendSlackNotification(unassignments) {
+      if (unassignments.length === 0) return;
+
+      try {
+        let message = "Automatically unassigned:\n";
+      unassignments.forEach(({ user, repo, issueNumber, owner }) => {
+      const issueUrl = `https://github.com/${owner}/${repo}/issues/${issueNumber}`;
+      message += `• '${user}' from <${issueUrl}|${repo}#${issueNumber}>\n`;
+    });
+
+        const response = await fetch(slackWebhookUrl, {
+          method: 'POST',
+          headers: {
+            'Content-Type': 'application/json'
+          },
+          body: JSON.stringify({ text: message })
+        });
+
+        if (!response.ok) {
+          throw new Error(`HTTP error! status: ${response.status}`);
+        }
+        console.log('Slack notification sent successfully');
+      } catch (error) {
+        console.error('Error sending Slack notification:', error);
+      }
+    }
+
+    const octokit = new Octokit({
```

Comment:
Using github actions script action would mean not having to worry about the authentication and setup piece here.